superset/db_engine_specs.py

```diff
@@ -104,8 +104,8 @@ def apply_limit_to_sql(cls, sql, limit, database):
             )
             return database.compile_sqla_query(qry)
         elif LimitMethod.FORCE_LIMIT:
-            sql_without_limit = cls.get_query_without_limit(sql)
-            return '{sql_without_limit} LIMIT {limit}'.format(**locals())
+            sql_before_limit, sql_after_limit = cls.get_query_without_limit(sql)
+            return '{sql_before_limit} LIMIT {limit}{sql_after_limit}'.format(**locals())
```

Comment:
What happens if one explicitly specifies a limit in the SQL which is less than the override?